analyzer/src/test/java/com/novoda/sqlite/impl/SQLFileTest.java

```diff
@@ -0,0 +1,93 @@
+package com.novoda.sqlite.impl;
+
+import java.io.IOException;
+import java.io.StringReader;
+import java.util.List;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertArrayEquals;
+import static org.junit.Assert.assertEquals;
+
+public class SQLFileTest {
+    private final static String STATEMENT_CREATE_TABLE = "CREATE TABLE banana(_id INTEGER);";
+    private final static String STATEMENT_ALTER_TABLE = "ALTER TABLE banana ADD COLUMN colour TEXT;";
+    private final static String NEW_LINE = "\n";
+    private final static String SPACE = " ";
```

Comment:
It'd have to be hardcoded multiple times in 4 tests (2 in the "given" part, 2 in the "assert" part). The assert part is a bit more unstable, since it verifies that there is only one space in a specific position.